docs/discussion/values/creativity-without-limits.rst

```diff
@@ -0,0 +1,56 @@
+Creativity Without Limits
+==============================
+
+Our focus on creativity without limits is about supporting users at all
+skill levels, and to help guide them from their first lines of code through
+contributing to open source.
```

Comment:
Is this really the end goal?

Reply:
It is explicitly part of the pitch I've been using in discussions with people.